pkgs/development/python-modules/bsddb3/default.nix

```diff
@@ -36,6 +36,17 @@ buildPythonPackage rec {
     export BERKELEYDB_DIR=${pkgs.db.dev};
   '';
 
+  postPatch = ''
+    substituteInPlace test3.py \
+      --replace-fail "from distutils.util import get_platform" "import platform" \
+      --replace-fail "get_platform()" "f'{platform.system().lower()}-{platform.machine()}'" \
+      --replace-fail "_TextTestResult" "TextTestResult"
```

Comment:
I no longer think it's a good idea to try to support python >=3.10.
- upstream explicitly says that it's not supported by them
- the testing code doesn't work on later python versions
- we're using `disabled = pythonAtLeast "3.10";`, so it would always require overriding the package

So if we don't care about >=3.10, we can drop this line:
```suggestion
```

Reply:
We could remove all the block of subsitute and replace-fail too no ?

Reply:
That other part was broken anyways, as it was not actually running any tests.

Reply:
Is it all good now ?